bulbs/api/views.py

```diff
@@ -75,7 +81,7 @@ class ContentViewSet(UncachedResponse, viewsets.ModelViewSet):
         "authors", "types"
     )
     permission_classes = [IsAdminUser, CanEditContent]
-    metadata_class = ContentViewMetaData
+    metadata_class = InfographicMetadataMixin
```

Comment:
This feels like it's making it more complicated, because we're adding a content-specific metadata class to a base view, without much explanation. 
Why not move some of the `InfographicMetadataMixin.determine_metadata()` logic into a restored `InfographicMetadataMixin.determine_metadata()` method, at least then you can document the ugliness directly inside that method.